### PR TITLE
Fixes #400 Document steps for generic installation

### LINKED ISSUE #400: Document steps for generic installation of Yaydoc
### This is a
- [ ] Bug report
- [ ] Feature request
- [x] Documentation request

### Description

Updating documentation by documenting steps for Yaydoc's generic installation would help new developers test and develop the application more effectively.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

issue #400 
## Motivation and Context

Updating documentation by documenting steps for Yaydoc's generic installation would help new developers test and develop the application more effectively.
## Types of changes

- [ ] Bug fix 
- [ ] New feature
- [x] Documentation update
## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by imujjwal96** [comment]
@pri22296 @sch00lb0y  Please review.

File: docs/installation/local.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Installing Yaydoc Locally
+
+## Prerequisites
+* NodeJS (v4.0 or above)
+* Python 2 or Python 3
```
Comment (pri22296) [inline docs/installation/local.md]: Python 2.7 or 3.3+

Comment (imujjwal96) [inline docs/installation/local.md]: :+1:

Comment (imujjwal96) [inline docs/installation/local.md]: updated

**Pull request merged**